java/ql/lib/semmle/code/java/frameworks/android/Intent.qll

```diff
@@ -52,3 +53,153 @@ class BundleGetterMethod extends Method, TaintPreservingCallable {
 
   override predicate returnsTaintFrom(int arg) { arg = -1 }
 }
+
+private class IntentBundleFlowSteps extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        //"namespace;type;subtypes;name;signature;ext;input;output;kind"
+        "android.os;BaseBundle;true;get;(String);;MapValue of Argument[-1];ReturnValue;value",
```

Comment:
Why do you model considerably more put than get methods? Looks like the numeric types have models for the setters but not the getters?

Reply:
All of the put methods carry flow to the MapKey, so all are relavent, whereas MapValue flow from getters (and the second argument of putters) is ommitted for types that are generally considere to not carry taint (i.e. numeric types).

Reply:
Aha, got it